conseil-api/src/main/resources/metadata/tezos.delphinet.conf

```diff
@@ -0,0 +1,1396 @@
+          visible: true
+        }
+        delegatable: {
+          visible: true
+        }
+        script: {
+          visible: true
+        }
+        storage: {
+          visible: true
+        }
+        status: {
+          visible: true
+          cache-config {
+            cached: true,
+            min-match-length: 4,
+            max-result-size: 100
+          }
+        }
+        consumed_gas: {
+          visible: true
+          display-name: "Consumed Gas"
+        }
+        block_hash: {
+          display-name: "Block Hash"
+          data-type: "hash"
+          visible: true
+          reference: {
+            entity: "blocks"
+            key: "hash"
+          }
+        }
+        block_level: {
+          visible: true
+          display-name: "Block Level"
+          reference: {
+            entity: "blocks"
+            key: "level"
+          }
+        }
+        timestamp: {
+          visible: true
+          data-format: "YYYY MMM DD, HH:mm"
+        }
+        internal {
+          visible: true
+        }
+        originated_contracts {
+          visible: true
+          display-name: "Originated Account"
+          data-type: "accountAddress"
+          reference {
+            entity: "accounts"
+            key: "account_id"
+          }
+          cache-config {
+            cached: true,
+            min-match-length: 4,
+            max-result-size: 100
+          }
+        }
+        storage_size {
+          display-name: "Storage Size"
+          visible: true
+        }
+        paid_storage_size_diff {
+          visible: true
+        }
+        ballot: {
+          visible: true
+          display-name: "Vote"
+        }
+        ballot_period: {
+          data-type: "int"
+          visible: true
+        }
+        number_of_slots: {
+          visible: true
+          display-name: "Slots"
+        }
+        branch: {
+          data-type: "hash"
+          visible: true
+        }
+        proposal: {
+          data-type: "hash"
+          visible: true
+        }
+        period: {
+          visible: true
+        }
+        errors: {
+          visible: true
+        }
+        utc_year: {
+          visible: true
+        }
+        utc_month: {
+          visible: true
+        }
+        utc_day: {
+          visible: true
+        }
+        utc_time: {
+          visible: true
+        }
+      }
+    }
+    bakers {
+      display-name-plural: "Bakers"
+      display-name: "Baker"
+      visible: true
+      attributes {
+        pkh {
+          display-name: "Address"
+          visible: true
+          data-type: "accountAddress"
+          reference: {
+            entity: "accounts"
+            key: "account_id"
+          }
+          cache-config {
+            cached: true,
+            min-match-length: 4,
+            max-result-size: 100
+          }
+          value-map: {
+            "tz1VxS7ff4YnZRs8b4mMP4WaMVpoQjuo1rjf": "Cryptonomic"
```

Comment:
Question same as for bakers_history: does it remains the same as in carthage?

Reply:
It does not. We should remove such entries.